net-misc/openconnect/files/openconnect-7.07-libressl.patch

```diff
@@ -0,0 +1,58 @@
+diff --git openssl-esp.c openssl-esp.c
```

Comment:
This patch is missing a descriptive header.

Please use the upstream commits as the basis for the patch(es). You can generate them using `git format-patch`, or by using the upstream gitweb patch function.

For example:
http://git.infradead.org/users/dwmw2/openconnect.git/patch/9c36560d3ed0aaf5d40c94fb18873584afe96cb8